todoapp/todos/serializers.py

```
@@ -1,5 +1,30 @@
+from todos import models as todos_models
+from users import serializers as users_serializers
 
-# Add your serializer(s) here
+
+class TodoSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = todos_models.Todo
+        fields = "__all__"
+
+    creator = users_serializers.UserSerializer(
+        exclude=["id", "date_joined"], source="user"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)